roles/openshift_node_group/files/sync.yaml

```diff
@@ -102,18 +102,30 @@ spec:
             ) &
             break
           done
-
+          mkdir /etc/origin/node/tmp
           # periodically refresh both node-config.yaml and relabel the node
           while true; do
-            if ! oc extract "configmaps/${name}" -n openshift-node --to=/etc/origin/node --confirm --request-timeout=10s --config /etc/origin/node/node.kubeconfig "--token=$( cat /var/run/secrets/kubernetes.io/serviceaccount/token )"  > /dev/null; then
+            if ! oc extract "configmaps/${name}" -n openshift-node --to=/etc/origin/node/tmp --confirm --request-timeout=10s --config /etc/origin/node/node.kubeconfig "--token=$( cat /var/run/secrets/kubernetes.io/serviceaccount/token )"  > /dev/null; then
               echo "error: Unable to retrieve latest config for node" 2>&1
               sleep 15 &
               wait $!
               continue
             fi
+
+            KUBELET_HOSTNAME_OVERRIDE=$(cat /etc/sysconfig/KUBELET_HOSTNAME_OVERRIDE) || :
```

Comment:
An additional file is not required - there is `/etc/sysconfig/origin-node`

Reply:
Easier to not have to muck with the existing file which may or may not be under some user's automation already.

Reply:
@michaelgugino Was there ever something that actually created this file or was it expected to be user provided?